src/development/tools/devtools/performance.md

```diff
@@ -18,28 +18,39 @@ your application. It consists of three parts, each increasing in granularity.
 * Timeline events chart
 * CPU profiler
 
-{{site.alert.note}}
+{{site.alert.secondary}}
   **Use a profile build of your application to analyze performance.**
-  Frame rendering times are not indicative of release performance
+  Frame rendering times aren't indicative of release performance
   unless your application is run in profile mode.
 {{site.alert.end}}
 
 The performance view also supports importing and exporting of
-data snapshots. For more information, see the [Import and export][] section.
+data snapshots. For more information,
+check out the [Import and export][] section.
+
+## What is a frame in Flutter?
+
+Flutter is designed to render its UI at 60 frames per second
+(fps), or 120 fps on devices capable of 120Hz updates.
+This means that, approximately every 16ms, the UI updates
+to reflect animations, or other changes to the UI. A frame
+that takes longer than 16ms to render can cause jank 
+(jerky movement).
 
 ## Flutter frames chart
 
-This chart contains Flutter frame information for your application. Each bar set
-in the chart represents a single Flutter frame. The bars are color-coded to
-highlight the different portions of work that occur when rendering a Flutter
-frame: work from the UI thread and work from the raster thread (previously known
+This chart contains Flutter frame information for your application.
+Each bar set in the chart represents a single Flutter frame.
+The bars are color-coded to highlight the different portions
+of work that occur when rendering a Flutter frame: work from
+the UI thread and work from the raster thread (previously known
 as the GPU thread).
```

Comment:
**suggestion:** Cut this sentence into two to make it more concise.
```suggestion
the UI and raster threads.
_Raster thread_ is the new term for the GPU thread.
```

Reply:
Again, it's not a "new" term. This change happened years ago. I leave it because it can still be useful for some.